platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/consensus/framework/validation/MiscValidation.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.test.consensus.framework.validation;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.swirlds.platform.internal.EventImpl;
+import com.swirlds.platform.test.consensus.framework.ConsensusOutput;
+import java.util.List;
+
+public class MiscValidation {
+    /**
+     * Verifies that the created round of new events does not advance when a quorum of nodes is
+     * down.
+     */
+    private static void createdRoundDoesNotAdvance(final ConsensusOutput output1, final ConsensusOutput ignored) {
```

Comment:
This is the only method in the class and it is private.  The IDE says the class is not used anywhere.   delete it?